course_discovery/apps/course_metadata/search_indexes.py

```diff
@@ -394,9 +394,9 @@ class PersonIndex(BaseIndex, indexes.Indexable):
     get_profile_image_url = indexes.CharField(model_attr='get_profile_image_url', null=True)
     position = indexes.MultiValueField()
     organizations = indexes.MultiValueField(faceted=True)
-    marketing_id = indexes.IntegerField(model_attr='marketing_id')
-    marketing_url = indexes.CharField(model_attr='marketing_url')
-    designation = indexes.CharField(model_attr='designation')
+    marketing_id = indexes.IntegerField(model_attr='marketing_id', null=True)
```

Comment:
How can the

Reply:
These won't be empty in any case but in the model, we allow those nulls these changes are to make fields consistent to the model.